code/Controllers/SilverStripeNavigatorItem_Unversioned.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SilverStripe\CMS\Controllers;
+
+use Config;
+use SilverStripe\CMS\Controllers\SilverStripeNavigatorItem;
+use SilverStripe\Core\Convert;
+use SilverStripe\Versioned\Versioned;
+
+class SilverStripeNavigatorItem_Unversioned extends SilverStripeNavigatorItem
+{
+    public function getHTML()
+    {
+        $recordLink = Convert::raw2att($this->getLink());
+        $linkTitle = _t('SilverStripe\\CMS\\Controllers\\ContentController.UNVERSIONEDPREVIEW', 'Preview');
+        return "<a class=\"current\" href=\"$recordLink\">$linkTitle</a>";
+    }
+
+    public function getLink()
+    {
+        return $this->getRecord()->PreviewLink();
+    }
+
+    public function getTitle()
+    {
+        return  _t(
+            'SilverStripe\\CMS\\Controllers\\ContentController.UNVERSIONEDPREVIEW',
+            'Preview',
+            'Used for the Switch between states (if any other other states are added). Needs to be a short label'
```

Comment:
What's this 3rd arg in `_t()` for?

Reply:
I just copied what was done for the existing `SilverStripeNavigatorItem`s but the PHPDoc in `i18n::t()` says "Any other string argument after default is context for i18nTextCollector"